instrumentation/README.md

```diff
@@ -23,7 +23,7 @@
 | [opentelemetry-instrumentation-mysql](./opentelemetry-instrumentation-mysql) | mysql-connector-python ~= 8.0 |
 | [opentelemetry-instrumentation-pika](./opentelemetry-instrumentation-pika) | pika >= 0.12.0 |
 | [opentelemetry-instrumentation-psycopg2](./opentelemetry-instrumentation-psycopg2) | psycopg2 >= 2.7.3.1 |
-| [opentelemetry-instrumentation-pymemcache](./opentelemetry-instrumentation-pymemcache) | pymemcache ~= 1.3 |
+| [opentelemetry-instrumentation-pymemcache](./opentelemetry-instrumentation-pymemcache) | pymemcache > 2, < 4 |
```

Comment:
Seems like this is dropping support for all pymemcache versions 1.3.X.

1. Looks like the PR title needs to be updated to reflect this
2. Is it possible to keep supporting versions 1.3.X and > 2, < 4?

Reply:
@ocelotl it was possible. I updated the code to support down to 1.3.5 (It didn't support below that before)